package.json

```diff
@@ -48,7 +49,7 @@
     "coveralls": "^3.0.0",
     "eslint": "^4.3.0",
     "eslint-config-google": "^0.9.1",
-    "ethereumjs-abi": "github:yondonfu/ethereumjs-abi#abi-offset-fixed-arrays",
```

Comment:
We were previously using my fork of this package with a specific fix. The fix was recently merged in, but a new release has not been cut for the NPM package yet, so for now we can install this dependency from Github using the commit hash that includes the mentioned fix